compose/cli/command.py

```diff
@@ -20,18 +21,19 @@
 def project_from_options(project_dir, options):
     return get_project(
         project_dir,
-        get_config_path_from_options(options),
+        get_config_path_from_options(project_dir, options),
         project_name=options.get('--project-name'),
         verbose=options.get('--verbose'),
     )
```

Comment:
It looks like all the calls to `Environment.from_env_file()` come from here. Instead of re-parsing the file multiple times, I think we could do it once here, and pass it through.